github/ci/prow/files/jobs/kubevirt/controller-lifecycle-operator-sdk/controller-lifecycle-operator-sdk-presubmits.yaml

```diff
@@ -0,0 +1,30 @@
+presubmits:
+  kubevirt/controller-lifecycle-operator-sdk:
+  - name: pull-controller-lifecycle-operator-sdk-e2e-k8s-1.18
+    always_run: true
+    optional: false
+    decorate: true
+    decoration_config:
+      timeout: 3h
+      grace_period: 5m
+    max_concurrency: 6
+    labels:
+      preset-dind-enabled: "true"
+      preset-docker-mirror: "true"
+      preset-shared-images: "true"
+    spec:
+      nodeSelector:
+        type: bare-metal-external
+      containers:
+        - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - "cd examples/sample-operator &&  export TARGET=k8s-1.18 && automation/test.sh"
+          # docker-in-docker needs privileged mode
```

Comment:
Does this job require docker in docker?

Reply:
Yes, there is a test suite based on kubevirtci.